doc/assessment-design.md

```diff
@@ -0,0 +1,126 @@
+
+The second question slot gives students one of `addVectors3`, `addVectors4`, or `addVectors5`. These are questions that the students haven't seen before, so we select from three question alternatives to minimize the risk of information transfer between students. Data shows that three or four question alternatives are normally sufficient ([Chen et al., 2018](http://lagrange.mechse.illinois.edu/pubs/ChWeZi2018a/)).
+
+The third and fourth questions test concepts that the students practiced on "Homework 1", but using different questions that were not on the homework. For this reason we again provide three question alternatives for each slot.
+
+```json
+{
+    "uuid": "424f3380-8e6d-42a2-8e72-f3c479f2711b",
+    "type": "Exam",
+    "title": "Vector algebra",
+    "set": "Quiz",
+    "number": "1",
+    "allowAccess": [
+        {
+            "role": "TA",
+            "credit": 100
+        },
+        {
+            "mode": "Exam",
+            "credit": 100,
+            "examUuid": "fa5e28c2-6733-47b7-a73f-5b58a21c8fb3",
+       }
+    ],
+    "zones": [
+        {
+            "title": "Fundamental questions",
+            "questions": [
+                {
+                    "numberChoose": 1,
+                    "points": [10, 9, 8, 7, 6],
+                    "alternatives": [
+                        {"id": "addVectors1"},
+                        {"id": "addVectors2"}
+                    ]
+                },
+                {
+                    "numberChoose": 1,
+                    "points": [10, 9, 8, 7, 6],
+                    "alternatives": [
+                        {"id": "addVectors3"},
+                        {"id": "addVectors4"},
+                        {"id": "addVectors5"}
+                    ]
+                }
+            ]
+        }
+        {
+            "title": "Advanced questions",
+            "questions": [
+                {
+                    "numberChoose": 1,
+                    "points": [5, 4, 4, 3, 3],
+                    "alternatives": [
+                        {"id": "dotProduct3"},
+                        {"id": "dotProduct4"},
+                        {"id": "dotProduct5"}
+                    ]
+                },
+                {
+                    "numberChoose": 1,
+                    "points": [5, 4, 4, 3, 3],
+                    "alternatives": [
+                        {"id": "crossProduct3"},
+                        {"id": "crossProduct4"},
+                        {"id": "crossProduct5"}
+                   ]
+                }
+            ]
+        }
+    },
+    "text": "For this quiz you can use the <a target=\"_blank\" href=\"<%= clientFilesCourse %>/formulas.pdf\">formula sheet</a>."
```

Comment:
Consider adding a note on escape character use in the text to explain why `\"` is present.